packages/cli/src/workflows/workflow.service.ts

```diff
@@ -137,6 +164,27 @@ export class WorkflowService {
 		});
 	}
 
+	private addWorkflowsCount(workflowsAndFolders: WorkflowAndFolderUnionFull[]) {
+		workflowsAndFolders.forEach((wf) => {
+			if (isFolder(wf)) {
+				wf.workflowsCount = wf?.workflows?.length ?? 0;
```

Comment:
It's a bit waste to select all the workflows for each folder in the query instead only their count. Something to consider in the future

Reply:
Agreed. This was me trying to predict the future. I thought we might need the workflowIds for some else in the folder's card and with this I could do both. Since we do not need anything like that just now, I deleted it and added the count directly.